Dockerfile

```diff
@@ -0,0 +1,22 @@
+# UI build
+FROM node:14-buster AS ui
```

Comment:
What is `buster`?

Reply:
the `node:14` tag has a pretty old version of `glibc` causing the node build to fail.